wgpu_shadertoy/shadertoy.py

```diff
@@ -70,7 +70,7 @@ class Shadertoy:
     Parameters:
         shader_code (str): The shader code to use.
         common (str): The common shaderpass code gets executed before all other shaderpasses (buffers/image/sound). Defaults to empty string.
-        resolution (tuple): The resolution of the shadertoy in (width, height). Defaults to (800, 450).
+        resolution (tuple): The resolution of the shadertoy in (width, height), will be changed by display scaling! Defaults to (800, 450).
```

Comment:
Maybe just say that it's the logical resolution, and maybe mention that the physical resolution may be different depending on the pixel ratio. I've found it to be helpful (for both devs and users) to always be explicit about what kind of size something is.